modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/npmscripts.config.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+module.exports = {};
```

Comment:
I suppose this file it's just a leftover, isn't? If not, why you leave this empty?

Reply:
If it's not useful i'll remove it right now